scripts/lua/rest/v1/edit/ntopng/user.lua

```diff
@@ -0,0 +1,91 @@
+--
+-- (C) 2013-20 - ntop.org
+--
+
+local dirs = ntop.getDirs()
+package.path = dirs.installdir .. "/scripts/lua/modules/?.lua;" .. package.path
+package.path = dirs.installdir .. "/scripts/lua/modules/pools/?.lua;" .. package.path
+
+require "lua_utils"
+local json = require ("dkjson")
+local rest_utils = require("rest_utils")
+
+--
+-- Edit a ntopng user
+-- Example: curl -u admin:admin -d '{"username": "mario", "user_role": "unprivileged", "allowed_interface": "", "allowed_networks": "0.0.0.0/0,::/0", "user_language": "en"}' http://localhost:3000/lua/rest/v1/edit/ntopng/user.lua
+--
+-- NOTE: in case of invalid login, no error is returned but redirected to login
+--
+
+sendHTTPHeader('application/json')
+
+local rc = rest_utils.consts_ok
+local res = {}
+
+if not haveAdminPrivileges() then
```

Comment:
Here we also could allow users to modify themselves (i.e. mario can modify mario), but this would complicate things as we need to prevent it from changing admin privileges (such as the interfaceId for instance). For now we'll leave it as is